packages/enzyme-test-suite/test/RSTTraversal-spec.jsx

```diff
@@ -43,6 +43,20 @@ describe('RSTTraversal', () => {
       const node = $(<div className={classes} />);
       expect(hasClassName(node, 'foo-bar')).to.equal(true);
     });
+
+    it('works if searching with a RegExp', () => {
+      function classes() {}
+      classes.toString = () => 'ComponentName-classname-123';
+      const node = $(<div className={classes} />);
```

Comment:
i'm really confused why we need a stringifiable function instead of just a string literal

Reply:
Copy from a more complex example.